src/components/ModalPin.js

```diff
@@ -101,14 +108,15 @@ export class ModalPin extends React.Component {
         </form>
       </div>
 
-    return <div>
+    return (
+      <div>
         <div className="modal fade" id="modalPin" tabIndex="-1" role="dialog" aria-labelledby="modalPin"
              aria-hidden="true">
           <div className="modal-dialog" role="document">
             <div className="modal-content">
               <div className="modal-header">
-                <h5 className="modal-title" id="exampleModalLabel">{t`Write your PIN`}</h5>
-                <button type="button" className="close" data-dismiss="modal" aria-label="Close">
+                <h5 className="modal-title" id="pinModal">{t`Write your PIN`}</h5>
```

Comment:
suggestion: This identifier may cause confusion with the modal div itself. How about `pinModalInputTitle`, or even remove the unique identifier altogether?

Reply:
Done! Thanks